anthos-bm-gcp-terraform/versions.tf

```diff
@@ -25,7 +25,7 @@ terraform {
     gcp_service_accounts  = "terraform-google-modules/service-accounts/google/v~> 4.0"
     gcp_compute_instances = "terraform-google-modules/vm/google//modules/compute_instance/v~> 6.3.0"
   }
-  required_version = "~> 0.15.5"
+  required_version = ">= 0.15.5"
```

Comment:
This should point to exact version `~> 1.x` used inside the docker image. Terraform have breaking changes even between minor versions.

Reply:
@Shabirmean Whats the reasoning behind this? With 1.0 the assumption is that TF will [no longer have breaking changes](https://www.terraform.io/docs/language/v1-compatibility-promises.html#the-terraform-language). Additionally the users can constrain which TF version they use while instantiating this module for more granular control.

Reply:
This is mainly because when the v1 of this sample was released there were breaking changes even for patch versions of terraform - `v0.14.9` and `v0.14.10`. So we wanted to make sure that reader following the sample has minimal to 0 friction and all information is explicit and tested.

They had a bug in an earlier version which resulted in not working for some of the people trying the sample

Reply:
set to the constraint we discussed offline